buildozer/targets/android.py

```diff
@@ -467,6 +471,10 @@ def _android_update_sdk(self, *sdkmanager_commands):
             command = '{} | {} --licenses'.format(
                 yes_command, self.sdkmanager_path)
             self.buildozer.cmd(command, cwd=self.android_sdk_dir)
+        if "CI" in environ:
+            sdkmanager_commands = list(sdkmanager_commands)
+            sdkmanager_commands.insert(len(sdkmanager_commands), '>/dev/null')  # mute stdout, important stuff goes to stderr anyways
+            sdkmanager_commands = tuple(sdkmanager_commands)
```

Comment:
why not skipping this whole part since you already introduced the `grep -v "^\\["` fix in the `_sdkmanager()` method above?